docs/zms_client.md

```diff
@@ -81,19 +81,13 @@ Use the `-z` option to point to the required environment for executing commands.
 ## How the ZMS Client Authenticates
 -----------------------------------
 
-The Athenz ZMS server requires the user to provide its `UserToken` to
-authorize the requests. The `UserToken` is obtained from the ZMS Server
+The Athenz ZMS server requires the user to authenticate using X509 certificate
```

Comment:
This is actually not quite correct. ZMS does not issue certificates and the user may not their own x.509 certificates (like we issue based on ssh credentials). So the UserToken is the correct reference here as the environment may use any type of Principal Authority to authorize the user.